rskj-core/src/main/java/org/ethereum/vm/PrecompiledContracts.java

```diff
@@ -108,6 +111,9 @@ public PrecompiledContract getContractForAddress(BlockchainConfig blockchainConf
         if (address.equals(IDENTITY_ADDR_DW)) {
             return identity;
         }
+        if (address.equals(BLOCK_HEADER_ADDR_DW)) {
```

Comment:
Do we maybe need to program this as a network upgrade?

Reply:
Added the logic to handle the new precompiled contract as a network upgrade. New RSKIP proposed: https://github.com/rsksmart/RSKIPs/pull/119